cmd/clusterctl/clusterdeployer/clusterclient/clusterclient.go

```diff
@@ -968,7 +972,17 @@ func waitForClusterResourceReady(cs clientset.Interface) error {
 }
 
 func waitForMachineReady(cs clientset.Interface, machine *clusterv1.Machine) error {
-	err := util.PollImmediate(retryIntervalResourceReady, timeoutMachineReady, func() (bool, error) {
+	timeout := timeoutMachineReady
```

Comment:
After second think, not sure if we can enhance the `clusterctl create` by adding a new flag to specify the `timeoutMachineReady`? 

Let's get some comments from @vincepri @detiber .

Reply:
In general, I would definitely prefer a flag to using environment variables, but if we extend this to other timeout values as well, then I would worry about how many flags would be needed to support the individual timeouts.

Reply:
Thanks @detiber  OK, let's stick on the environment variable.